pkg/sdn/plugin/node_iptables.go

```diff
@@ -95,6 +95,7 @@ func (n *NodeIPTables) getStaticNodeIPTablesRules() []FirewallRule {
 		{"filter", "INPUT", []string{"-p", "udp", "-m", "multiport", "--dports", VXLAN_PORT, "-m", "comment", "--comment", "001 vxlan incoming", "-j", "ACCEPT"}},
 		{"filter", "INPUT", []string{"-i", TUN, "-m", "comment", "--comment", "traffic from SDN", "-j", "ACCEPT"}},
 		{"filter", "INPUT", []string{"-i", "docker0", "-m", "comment", "--comment", "traffic from docker", "-j", "ACCEPT"}},
+		{"filter", "FORWARD", []string{"-s", n.clusterNetworkCIDR, "-m", "conntrack", "--ctstate", "INVALID", "-j", "DROP"}},
```

Comment:
I think the fact that this is needed means our rules are otherwise broken; there shouldn't be any rule that would allow non-masqueraded traffic to be forwarded, so it ought to get dropped by default.

Maybe the "FORWARD -s clusterNetworkCIDR" rule just shouldn't be there... what happens if you remove it?